wordpress-mobile/WordPress-Android
# QR Code Auth: Part 2 - Layouts, styles, strings, and illustrations

Parent #16481 

This PR adds the following for the QRCodeAuth flow
- Layout files for loading, content, and error views
- Update qrcodeauth_fragment to include the above layouts files
- Adds qrcodeauth_styles
- Adds string resources

Notes: 
- This PR will be merged to a feature branch and not trunk. Milestone will be set to "future"
- Not all strings declared are used in this PR, but they will be before the feature branch is requested to be merged in to trunk.

**Merge Instructions**
- Ensure that #16706 has been merged to the feature branch
- Merge as normal

**To test:**
- Install the app
- Login using a WP.com account
- Navigate to Me -> App Settings -> Debug Settings
- Enable `QRCodeAuthFlowFeatureConfig`
- Restart the App
- Navigate back to Me
-Tap the "Scan Login Code" row item
- The view should scroll through the above layouts - (You will not see the entire view, just the progress indicator and the buttons)

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
